erts/emulator/test/op_SUITE.erl

```diff
@@ -424,6 +425,20 @@ eval(E0) ->
         {value,Val,_Bs} -> Val
     end.
 
+unsafe_fusing(_Config) ->
+    {0,0,1} = usec_to_time(id(1)),
+    {1,234_567,890_000} = usec_to_time(id(1_234_567_890*1_000)),
+    ok.
+
+usec_to_time(Usec) ->
+    Us = Usec rem 1000000,
+
+    %% div and rem would be fused, which is wrong since rem should use
+    %% the result of the div.
+    Sec = (Usec div 1000000) rem 1000000,
+    MegaSec = (Usec div 1000000) div 1000000,
+    {MegaSec, Sec, Us}.
+
```

Comment:
Can it be simplified to just `(Usec div 1000000) rem 1000000`?
(Us and MegaSec don't seem to make a difference).

Reply:
Yes, it can be simplified. I also added a better explanation of what caused the bug.